components/dashboards/org.wso2.carbon.dashboards.core/src/main/java/org/wso2/carbon/dashboards/core/bean/widget/WidgetConfigs.java

```diff
@@ -1,25 +1,30 @@
 /*
- *  Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Can't change the year of the license header please revert back

Reply:
fixed with 87b573730d4c8021f11a7fc89783b583eed6edbf